src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -2875,6 +2875,41 @@ constexpr char const CORE_RPC_STATUS_TX_LONG_POLL_MAX_CONNECTIONS[] = "Daemon ma
     typedef epee::misc_utils::struct_init<response_t> response;
   };
 
+  LOKI_RPC_DOC_INTROSPECT
+  // Get information on Service Node.
+  struct COMMAND_RPC_GET_SERVICE_NODE_STATUS
+  {
+    struct request_t
+    {
+      bool include_json;                             // When set, the response's as_json member is filled out.
+
+      BEGIN_KV_SERIALIZE_MAP()
+        KV_SERIALIZE(include_json);
+      END_KV_SERIALIZE_MAP()
+    };
+    typedef epee::misc_utils::struct_init<request_t> request;
+
+    struct response_t
+    {
+
+      struct cryptonote::COMMAND_RPC_GET_SERVICE_NODES::response_t::entry service_node_state; // Service node registration information
```

Comment:
You don't need `struct` here (it's a C thing in this context that isn't needed at all in C++).